### PR TITLE
remove lodash

### LINKED ISSUE #3: remove lodash in favor of ramda

### PR DESCRIPTION
**Pull request by aliceriot**
this removes lodash 

fixes #3 

unfortunately though, it looks like lodash is a dependency of a lot of our dependencies, so it might still get pulled in when installing.

### CODE REVIEW

File: src/util.js [hunk 1]
```diff
@@ -18,5 +17,5 @@ export const withUsername = (type: ActionType, payloadFunc: Function = R.nthArg(
 )
 
 export const updateStateByUsername = (state: Object, username: string, update: Object) => (
-  _.merge({}, state, { [username]: update })
+  Object.assign({}, state, { [username]: update })
```
Comment (noisecapella) [inline src/util.js]: Can we use the spread syntax here instead?

**Comment by aliceriot** [comment]
I changed that and added a few tests as well

**Comment by aliceriot** [comment]
I'll go ahead and merge if you think that looks good!

**Comment by noisecapella** [comment]
:+1:

**Comment by aliceriot** [comment]
I just realized I forgot one thing, I'll ping you when I address it

**Comment by aliceriot** [comment]
I forgot we were using `_.merge` so that updates would be merged into the existing namespaced (per-username) state. So I added a test to make sure that's happening, and added the `lodash.merge` package so we can get just that method.

**Comment by aliceriot** [comment]
@noisecapella the changes are pretty trivial but if you wanna take a look before I merge that would be :100:

**Pull request merged**